frontend/package.json

```diff
@@ -20,13 +20,15 @@
     "@types/react-redux": "^7.1.27",
     "axios": "^1.5.1",
     "bootstrap": "^5.3.2",
+    "express": "^4.18.2",
     "framer-motion": "^10.16.4",
     "i": "^0.3.7",
     "npm": "^10.2.1",
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.0",
     "react-cookie": "^6.1.1",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.11.0",
     "react-markdown": "^9.0.0",
```

Comment:
did you install these libraries on the frontend by accident?

`express` doesnt seem needed for the frontend
did you use icons from the `react-icons` library?